modules/installation-creating-aws-control-plane.adoc

```diff
@@ -6,9 +6,14 @@
 [id="installation-creating-aws-control-plane_{context}"]
 = Creating the control plane machines in AWS
 
-You must create the control plane machines in Amazon Web Services (AWS) for your
-cluster to use. The easiest way to create these nodes is
-to modify the provided CloudFormation template.
+You must create the control plane machines in Amazon Web Services (AWS) for your cluster to use.
+
+You can use the provided CloudFormation template and a custom parameter file, to create a stack of AWS resources that represent the control plane nodes.
```

Comment:
s/file, to/file to
Alternatively, here and elsewhere you could swap the sentence order to say:
"To create a stack of AWS resources that represent the control plane nodes, you can use the provided CloudFormation template and a custom parameter file."

Reply:
Thank you, I have removed the comma. On review I would like to keep the sentence order as it stands in this case.